svelte-case-study.md

```diff
@@ -0,0 +1,203 @@
+tags: Development Platforms, Svelte, JavaScript framework
+sidebar: development-platforms
+permalink: development-platforms/svelte.html
+folder: development-platforms
+---
+
+## Svelte - the magical disappearing UI framework
+
+Svelte is a free and open-source front-end component framework in the JavaScript ecosystem. Created by Rich Harris in 2016, it has quickly become one of the most popular JavaScript frameworks available. Svelte differs from other JavaScript frameworks, mainly in how it ships components to the browser. Unlike React and Vue, which render components in a virtual DOM, Svelte compiles HTML templates into specialized, highly optimized code at build time, resulting in improved performance.
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/30121693/217351206-6f367e60-d963-40a4-81f2-2f4ec54b910f.png" alt="Svelte messiah meme"/>
+</p>
+
+This case study aims to introduce the reader to Svelte as a development tool for building front-end applications. It does not aim to make the reader an expert in the framework by the end, but rather to provide the reader with the necessary information to form an informed opinion on whether or not Svelte will be a good development platform for them.
+
+## Brief History
+
+Released in November 2016, Svelte started out as an improved version of its predecessor, Ractive.js, with the addition of a compiler. Ractive.js was created at The Guardian as an in-house tool, but eventually, Rich Harris took it to the next level and released it to the world as Svelte. In April 2018, the developers released the second version to correct what they considered to be mistakes in the previous build. The major turning point came in April 2019 with the release of version 3. Svelte was re-built using TypeScript and had a completely rethought reactivity system, using its compiler to handle the heavy lifting behind the scenes. Svelte, like a magician who leads you to focus on a specific part of the show instead of the set-up of the trick, handles component loading and unloading out of view of the user instead of a virtual DOM.
+
+The extension, SvelteKit, was announced in October 2020 and entered beta in March 2021. SvelteKit aims to be a more complete, or all-in-one, service. It is designed to make it easier to build fast, modern web applications by offering a set of tools for building, deploying, and scaling applications with Svelte. SvelteKit provides a standard architecture for building web applications and makes it easier to handle common tasks such as routing, code splitting, and asset handling. It also provides a set of features for building server-side rendering applications and integrates with modern tools such as webpack and Rollup. SvelteKit is designed to work seamlessly with the Svelte JavaScript framework and provides a familiar and consistent development experience for Svelte developers.
+
+## Features
+
+The key features of Svelte are:
+
+- No virtual DOM
```

Comment:
Consider using a markdown table here since you have one heading and one subpoint each

Reply:
I tried setting up a table, but I don't really feel like it's more organized or looks better. If I had more subpoints, it might fit better in my mind. I think I'll leave it as-is.